completion/bash/task.bash

```diff
@@ -1,26 +1,48 @@
-#!/bin/bash
+# vim: set tabstop=4 shiftwidth=4 expandtab:
 
-GO_TASK_PROGNAME=task
```

Comment:
AFAIK this variable was here to allow someone to have completion even if the Task binary is named differently.

Due to a conflict with taskwarrior, some installation methods may call it `go-task` for example.

Don't you think it's worth keeping it?

Reply:
Within completion functions you better use `$1` or the array variable `$COMP_WORDS` to get the actual command the completion is running for. There's no need for a global variable. This is what the code does.

The line
```sh
complete -F _task task
````

attaches the completion function to a program named `task` (the last word).

It may make some sense to allow customization here. However, the previous approach was also useless, because:
1. The variable `GO_TASK_PROGNAME` was set unconditionally. How can any user change it between the beginning and the end of the script?
2. The bash completion framework loads completions on demand matching the command name `task` with the file name `task.bash`. This doesn't work if the binary is called differently, but the script still has the old name. It always requires more changes which those creating packages can provide easily (rename the file and change the last line with a regex).

I remove it because it wasn't really working and I didn't expect that use case.